.github/workflows/ossf-scorecard-reporting.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         with:
           egress-policy: audit # TODO: change to 'egress-policy: block' after couple of runs
 
-      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v3.3.0
+      - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v3.3.0
```

Comment:
Why did the SHA change but the version didn't change?

The SHA before and after don't match the most recent commit on https://github.com/actions/checkout/commits/v3.3.0

See: ac593985615ec2ede58e132d2e21d2b1cbd6127c